src/corelibs/U2Gui/src/OpenViewTask.cpp

```diff
@@ -84,13 +84,16 @@ static Task* createOpenViewTask(const MultiGSelection& ms) {
             ls.removeAll(f);
         }
     }
-
+    Task* t = nullptr;
+    if (QApplication::activeModalWidget() != nullptr) {
+        coreLog.info(QObject::tr("Unable to open view because of active modal widget."));
```

Comment:
This is a big limitation and we should allow creation of views in the background.
May be it worth to create a separate bug for that.